services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/paatos/PaatosServiceITest.kt

```diff
@@ -100,4 +135,109 @@ class PaatosServiceITest(
             assertThat(result).extracting { it.hakemusId }.containsExactly(hakemus2.id)
         }
     }
+
+    @Nested
+    inner class SaveKaivuilmoituksenPaatos {
+        private val eventTime = ZonedDateTime.parse("2024-07-21T15:23:12Z")
+        private val alluId = 631
+        private val hakemustunnus = "KP2400414"
+        private val nameFromAllu = "Name from Allu"
+        private val startDateFromAllu = ZonedDateTime.parse("2024-07-24T15:23:12Z")
+        private val endDateFromAllu = ZonedDateTime.parse("2024-07-25T15:23:12Z")
+        private val event =
+            ApplicationStatusEvent(
+                eventTime = eventTime,
+                applicationIdentifier = hakemustunnus,
+                newStatus = ApplicationStatus.DECISION,
+                targetStatus = null,
+            )
+
+        private fun setupAlluMocks() {
+            every { alluClient.getDecisionPdf(alluId) } returns DEFAULT_DATA
```

Comment:
Should we rename this `DEFAULT_DATA` to something like `PAATOS_PDF_CONTENT` or such?

Reply:
I renamed this to `PDF_BYTES`.